to.us.scrollprotocol.Profectus.yml

```diff
@@ -0,0 +1,49 @@
+id: to.us.scrollprotocol.Profectus
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: profectus
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-download:rw
+  # Audio
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+modules:
+  - name: sdl2-gfx
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/sdl2gfx/files/SDL2_gfx-1.0.4.tar.gz/download
+        sha256: 63e0e01addedc9df2f85b93a248f06e8a04affa014a835c2ea34bfe34e576262
+        archive-type: tar-gzip
+      - type: shell
+        commands:
+          - cp /usr/share/automake-1.16/{config.guess,config.sub} .
+  - name: profectus
+    buildsystem: simple
+    build-commands:
+      - CGO_CFLAGS="`pkg-config --cflags SDL2_gfx` -DSDL_DISABLE_IMMINTRIN_H" CGO_LDFLAGS=`pkg-config --libs SDL2_gfx` $GOROOT/bin/go build -trimpath -o profectus
```

Comment:
if you notice the build failure wasn't in this module. I don't think this change is relevant in any way.

Reply:
Ok, thanks.